.github/workflows/unittest.yml

```diff
@@ -25,23 +27,9 @@ jobs:
           python-version: ${{ matrix.python-version }}
       - name: Install dependencies
         run: |
-          pip install tensorflow==${{matrix.tf-version}}
+          pip install tensorflow-cpu==${{matrix.tf-version}} || pip install tensorflow==${{matrix.tf-version}}
```

Comment:
This change is just to speedup CI if a CPU only version of TensorFlow is available.